### PR TITLE
[ci] switch from Ubuntu 12.04 to 14.04 on travis-ci.org

### PR DESCRIPTION
**Pull request by adrianschroeter**
Ubuntu 12.04 is going out of maintenance soon.

### CODE REVIEW

**Comment by adrianschroeter** [comment]
seems we need to package a proper version of yum and createrepo for 14.04 first ...

**Comment by bgeuken** [comment]
@adrianschroeter Is that something you are going to do? Otherwise it might be something for the product backlog

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,8 @@ before_install: dist/ci/travis_before_install.sh
 before_script: dist/ci/travis_before_script.sh
 after_failure: dist/ci/travis_after_failure.sh
 script: "dist/ci/travis_script.sh $TEST_SUITE"
+dist: trusty
+sudo: required
```
Comment (mdeniz) [inline .travis.yml]: This line is already at the end of the file. So, it's duplicated.

**Comment by bgeuken** [comment]
Closing for now. Feel free to reopen once there is an update on this one

**Comment by adrianschroeter** [comment]
I haven't time for it yet, but anyone else can do the packaging as well for Ubuntu 14.04

**Comment by bgeuken** [comment]
What's the point in having this open? It's just a dead PR

**Comment by adrianschroeter** [comment]
it is not dead, we need to move to new ubuntu, since travis will switch some day. The current one is EOL.

**Comment by hennevogel** [comment]
[Update travis base distro](https://trello.com/c/8UXUIQgL/284-update-travis-base-distro)

**Comment by hennevogel** [comment]
I've added a card to the backlog. That is where we track things that we want to do in the future...

**Pull request closed without merging**